src/Microsoft.AspNet.Mvc.Xml/Formatters/XmlDataContractSerializerInputFormatter.cs

```diff
@@ -106,7 +126,7 @@ protected virtual XmlReader CreateXmlReader([NotNull] Stream readStream)
         /// <returns>The <see cref="XmlObjectSerializer"/> used during deserialization.</returns>
         protected virtual XmlObjectSerializer CreateDataContractSerializer(Type type)
         {
-            return new DataContractSerializer(SerializableErrorWrapper.CreateSerializableType(type));
```

Comment:
Should SerializableErrorWrapper.CreateSerializableType be moved to ReadInternal? It is not present in XmlDataContractSerializerOutputFormatter.CreateSerializer.  So the behavior is inconsistent between the input and output formatters.